src/main/java/org/broadinstitute/hellbender/tools/exome/CalculateTargetCoverage.java

```diff
@@ -34,12 +35,42 @@
  * Calculate read-counts across targets given their reference coordinates.
  *
  * @author Valentin Ruano-Rubio &lt;valentin@broadinstitute.org&gt;
+ *
+ * <h3>Examples</h3>
+ *
+ * <p>
+ *     The command encompasses empirically determined parameters for TCGA project data.
+ *     You may obtain better results with different parameters.
```

Comment:
This might not be clear to some users.  You should make clear that this tool is an input to our exome CNV pipeline, and so different parameters may give better results *for exome CNV calling*.